Readability.js

```diff
@@ -600,7 +600,7 @@ Readability.prototype = {
 
       // If the resulting title is too short, remove the first part instead:
       if (wordCount(curTitle) < 3) {
-        curTitle = origTitle.replace(/^[^\|\-\\\/>»]*[\|\-\\\/>»]/gi, "");
+        curTitle = origTitle.replace(/^[^\|\-\–\—\\\/>»]*[\|\-\\\/>»]/gi, "");
```

Comment:
I think we also need/want to update the regexes a few lines above this (the `.test` and `.matchAll` calls), right?

Reply:
Yes, sorry for the oversight! Should be fixed. I'm also suggesting to move the separators into a separate expression to avoid making this mistake in the future (see my message below).